src/components/AirdropWalletsCarouselItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { CONST, AirdropWalletDetails, AirdropGoal } from 'app/consts';
+
+import { AirdropWalletBalance } from './AirdropWalletBalance';
+import { AirdropWalletFooter } from './AirdropWalletFooter';
+
+const i18n = require('../../loc');
+
+const airdropGoals: AirdropGoal[] = [
+  { threshold: 5, name: i18n.airdrop.walletsCarousel.shrimp },
+  { threshold: 25, name: i18n.airdrop.walletsCarousel.crab },
+  { threshold: 100, name: i18n.airdrop.walletsCarousel.shark },
+  { threshold: 1000, name: i18n.airdrop.walletsCarousel.whale },
+];
+
+const readableOrder = [i18n.order.first, i18n.order.second, i18n.order.third, i18n.order.fourth];
+
+interface Props {
+  walletDetails: AirdropWalletDetails;
+}
+
+export const AirdropWalletsCarouselItem: FC<Props> = ({ walletDetails }) => {
+  const unreachedGoals = airdropGoals.filter((goal: AirdropGoal) => goal.threshold > walletDetails.balance);
+  // TODO: edge case of "all goals reached" not covered by designs. Awaiting UX input. For now returning last one - "whale"
+  const nextGoal = unreachedGoals.length > 0 ? unreachedGoals[0] : airdropGoals[airdropGoals.length - 1];
+  const nextGoalIndex = airdropGoals.findIndex((goal: AirdropGoal) => goal.threshold === nextGoal.threshold);
+
+  return (
+    <View style={styles.walletCard}>
+      <AirdropWalletBalance
+        balance={walletDetails.balance}
+        walletName={walletDetails.name}
+        threshold={nextGoal.threshold}
+        footer={
```

Comment:
I have doubts if extracting `AirdropWalletFooter` in this case simplified things or unnecessarily increased complexity 🤔

Reply:
this footer will be used as soon as we get to implement 'airdrop finished' screen - text in footers is different there, that s why it s a component with props.

Reply:
I would also keep the footer here, not in a separate component. The AirdropWalletsCarouselItem (please change name comment before) should be reused in the airdrop finished, so the AirdropWalletFooter won't be used again

Reply:
ah, true, went level too deep, whole component will be reused. Changing

Reply:
done